pkg/controllers/routing/ecmp_vip.go

```diff
@@ -65,7 +66,7 @@ func (nrc *NetworkRoutingController) newServiceEventHandler() cache.ResourceEven
 			nrc.OnServiceUpdate(obj)
 		},
 		UpdateFunc: func(oldObj, newObj interface{}) {
-			nrc.OnServiceUpdate(newObj)
+			nrc.OnServiceUpdate(newObj, oldObj)
```

Comment:
@mk01 Please add new function `OnServiceCreate` and use it to handle `AddFunc`. And modify the signature of `OnServiceUpdate` to take the new and old objects explicitly (instead of objOld ...interface{})

There should have been separate handler for just Add to begin with. Nevertheless its good to modify it now.